backend/util/verifyRoles.js

```diff
@@ -0,0 +1,8 @@
+const verifyRoles = (allowedRole) => {
+    return (req, res, next) => {
+        if(!allowedRole === req.roles) return res.sendStatus(401);
```

Comment:
`if (!allowedRole === req.roles)` means that user doesn't have enough permission to access resource, so it is `403`